.travis.yml

```diff
@@ -79,6 +79,10 @@ script:
       presto-product-tests/bin/run_on_docker.sh \
```

Comment:
Could you please separate this commit into two. First create an environment called `multinode-tls-kerberos`, but without kerberos support. Just with HTTPS communication enabled. And then in a separate commit enable kerberos for this enviroment. It will be much easier to track and to review.